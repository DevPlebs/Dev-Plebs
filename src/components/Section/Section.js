import styled from "@emotion/styled"
import tw from "tailwind.macro"

export const Section = styled.div`
  ${tw`relative pb-16 sm:pb-18 md:pb-20 pt-16 sm:pt-18 md:pt-20`}
  background-color: ${props => props.bgColor};
  min-height: 525px;
  svg {
    top: 0;
  }
`

Section.defaultProps = {
  bgColor: "#fff",
}

export default Section
