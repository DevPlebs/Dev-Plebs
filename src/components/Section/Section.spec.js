import React from "react"
import { create } from "react-test-renderer"

import Section from "./Section"

describe("Section", () => {
  it("renders correctly", () => {
    const tree = create(<Section />)
    expect(tree.toJSON()).toMatchSnapshot()
  })
})
