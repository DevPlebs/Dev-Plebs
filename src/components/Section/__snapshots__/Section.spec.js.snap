// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Section renders correctly 1`] = `
.emotion-0 {
  position: relative;
  padding-bottom: 4rem;
  padding-top: 4rem;
  background-color: #fff;
  min-height: 525px;
}

@media (min-width:768px) {
  .emotion-0 {
    padding-bottom: 5rem;
    padding-top: 5rem;
  }
}

.emotion-0 svg {
  top: 0;
}

<div
  className="emotion-0"
/>
`;
